server/models/EmailTemplateModel.py

```diff
@@ -9,16 +16,42 @@ class EmailTemplateModel(db.Model, SerializerMixin):
     name = db.Column(db.String(120))
     type = db.Column(db.String(120))
     subject = db.Column(db.String(120))
-    body = db.Column(db.VARCHAR(2000)) 
+    body = db.Column(db.VARCHAR(500)) 
```

Comment:
Isn't 500 char might be too little, hence I put it to 2000, could be changed accordingly